### DISCUSSION TITLE
VGA is working!

### DISCUSSION THREAD

**Discussion by ulften** [picocomputer/community#2] [Q&A]
Hi!
I thought that I should share my progress with my version of Rumbledethumbs' picocomputer, which I have been following with great interest and admiration. 

After a great start getting the hello_world and even Basic working, I had a bit of a struggle with Advent4. I got it going in the end and it was great to see that we now have a file system accessible from the 6502 as well. Since, I have used the VGA demo board in my design as a way to  get access to the  VGA resistor network, d-sub etc., I was eager to try out VGA as well. After the trouble with the text based Advent4, I had anticipated even more trouble with VGA, but it turned out to be pretty smooth sailing. The screen remained black at first, so I needed to hook up an oscilloscope to verify the sync signals and then realized that I had them cross wired going from the VGA pico to the Demo board. Then the screen came to life. I have posted a few pictures, showing my board and my latest Youtube hero!! :-)
![VGAworking2](https://user-images.githubusercontent.com/129115806/231698624-e13e9667-2939-44bd-8662-78c5587066c5.jpg)
![VGAworking1](https://user-images.githubusercontent.com/129115806/231698655-8936727a-ac31-4ae9-9ece-411a066398ee.jpg)

I have enjoyed this project very much, and will now need to think about how to extend it and move forward from here!
Ulf

**Comment by larryvc** [comment]
Kudos on the great work Ulf!   I echo your statement above regarding Rumbledethumps and this project, it is the first in a very long time to pique my interest.

I just finished my board build yesterday, sort of followed your layout.   Still getting up to speed with the software and just now managed to get hello world to run.  My thoughts about moving on from here would include putting this on a circuit board, extending the IO capabilities, but mostly, just having fun with it.

**Comment by ulften** [comment]
Thanks!
